include/experimental/__p0009_bits/layout_stride.hpp

```diff
@@ -657,7 +657,7 @@ constexpr void validate_strides(with_rank<N>, Layout, const Extents& ext, const
     const std::size_t s = is_left ? r : N - 1 - r;
 
     MDSPAN_IMPL_PRECONDITION(common_integral_compare(expected_stride, other.stride(s))
-                             and "invalid strides for layout_{left,right}");
+                             && "invalid strides for layout_{left,right}");
```

Comment:
Just to make sure, do we intentionnally apply this binary AND between a boolean and a c-string ?

```cpp
MDSPAN_IMPL_PRECONDITION(common_integral_compare(expected_stride, other.stride(s)) && "invalid strides for layout_{left,right}")
```

Reply:
Yeah its a trick to get nice messages ...